### ISSUE TITLE
Ensure focus-trap-js is tested

### ISSUE THREAD

**Issue by lychyi** [Workday/canvas-kit#391]
We depend on a focus trap library that is maintained by a third-party person. This code is shipped with our modal and we should have it completely covered by unit tests. This is a request here: https://github.com/alexandrzavalii/focus-trap-js/issues/3

I'm opening this issue to track the above. An alternative is that we can fork the code and test in our own repo. It's not very large at all.

**Comment by NicholasBoll** [comment]
Closed byhttps://github.com/alexandrzavalii/focus-trap-js/pull/4
